app/src/main/java/com/infomaniak/mail/ui/main/menu/ManageMailAddressFragment.kt

```diff
@@ -25,31 +25,25 @@ import androidx.core.content.res.ResourcesCompat
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import androidx.lifecycle.lifecycleScope
-import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.infomaniak.lib.core.views.DividerItemDecorator
 import com.infomaniak.mail.R
-import com.infomaniak.mail.databinding.DialogConfirmLogoutBinding
+import com.infomaniak.mail.databinding.DialogWithDescriptionBinding
 import com.infomaniak.mail.databinding.FragmentManageMailAddressBinding
 import com.infomaniak.mail.ui.main.user.ManageMailAddressViewModel
 import com.infomaniak.mail.ui.main.user.SimpleMailboxAdapter
 import com.infomaniak.mail.utils.AccountUtils
 import com.infomaniak.mail.utils.animatedNavigation
+import com.infomaniak.mail.utils.createAlert
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 
 class ManageMailAddressFragment : Fragment() {
 
     private lateinit var binding: FragmentManageMailAddressBinding
-    private val dialogBinding by lazy { DialogConfirmLogoutBinding.inflate(layoutInflater) }
+    private val dialogBinding by lazy { DialogWithDescriptionBinding.inflate(layoutInflater) }
```

Comment:
I'd shorten the name with something like DescriptionDialog to avoid using the word "with"

Reply:
`DescriptionDialogBinding`